src/Parcel/Parcel.spec.js

```diff
@@ -106,19 +107,55 @@ describe('Parcel', function() {
   describe('.findWithLastActiveMortgageByBorrower', function() {
     beforeEach(() => new ParcelService().insertMatrix(0, 0, 10, 10))
 
-    it('should return parcels with mortgages open/claimed by borrower', async function() {
+    it('should return parcels with mortgages by borrower', async function() {
+      const publication = {
+        tx_hash: '0xdeadbeef',
+        tx_status: txUtils.TRANSACTION_STATUS.confirmed,
+        status: Publication.STATUS.open,
+        type: ASSET_TYPE.parcel,
+        asset_id: '2,5',
+        owner: '0xdeadbeef33',
+        buyer: null,
+        price: 1500,
+        expires_at: new Date().getTime() * 1000,
+        contract_id: '0xdeadbeef',
+        block_time_created_at: null,
+        block_time_updated_at: null,
+        marketplace_id: '0x113322',
+        block_number: 1
+      }
+      const publication2 = {
+        tx_hash: '0xdeadabeef',
+        tx_status: txUtils.TRANSACTION_STATUS.confirmed,
+        status: Publication.STATUS.open,
+        type: ASSET_TYPE.parcel,
+        asset_id: '6,5',
+        owner: '0xdeadbeef33',
+        buyer: null,
+        price: 1500,
+        expires_at: new Date().getTime() * 1000,
+        contract_id: '0xdseadbeef',
+        block_time_created_at: null,
+        block_time_updated_at: null,
+        marketplace_id: '0x113322',
+        block_number: 1
+      }
+      await Publication.insert(publication)
+      await Publication.insert(publication2)
```

Comment:
Can we run this two with `Promise.all(<PromisesArray>)` since we are not assigning them to variables we can take advantage of asynchronism right?